inductiva/scenarios.py

```diff
@@ -1,11 +1,13 @@
 """Base class for scenarios."""
 
-from abc import ABC, abstractmethod
+from abc import ABC
```

Comment:
If it's possible, can you also, pick up and fix some of the imports, please? We still have a lot of class imports in these files.